### PR TITLE
1042 create card child component

### PR DESCRIPTION
**Pull request by stackseeder**
Screenshot from 2021-09-01 12-49-20

Test on:
https://pr581--gpui.review.gnosisdev.com/storybook/?path=/story/common-cardrow--default

### CODE REVIEW

File: src/components/common/Card/cardContent.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  hint?: string
+  status?: statusType
+}
+
+/**
+ * Card component.
+ *
+ * An extensible content container.
+ */
+export const CardContent: React.FC<CardContentProps> = ({ label, value, hint, status, ...rest }): JSX.Element => {
```
Comment (alfetopito) [inline src/components/common/Card/cardContent.tsx:50]: Make the file name also PascalCase like the component

**Review by alfetopito — COMMENTED** [review]
One minor comment regard file name.

Otherwise, looks good!

I'll ask @alongoni and @biocom to review regarding the CSS though, as I feel like the texts are not centered, but I'm no qualified to make them so.

**Comment by alongoni** [comment]
> One minor comment regard file name.
> 
> Otherwise, looks good!
> 
> I'll ask @alongoni and @biocom to review regarding the CSS though, as I feel like the texts are not centered, but I'm no qualified to make them so.

Maybe we can center the texts using:
```
    display: flex;
    align-items: center;
    justify-content: center;
```
I'll try to fix that with Ivica-

File: src/components/common/Card/cardContent.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import styled from 'styled-components'
+
+const CardBody = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  p {
+    font-size: 15px;
+    margin-bottom: 0px;
```
Comment (alongoni) [inline src/components/common/Card/cardContent.tsx:11]: `margin: 0;
`
In order to center the content.

**Review by alongoni — CHANGES_REQUESTED** [review]

File: src/components/common/Card/cardContent.tsx (hunk 1 above)
Comment (alfetopito) [reply src/components/common/Card/cardContent.tsx:49]: Still not resolved :)

File: src/components/common/CardRow/CardRow.stories.tsx [hunk 3]
```diff
@@ -17,16 +20,40 @@ const Template: Story<CardRowProps> = (args) => (
...
+          value1="511.12k"
+          caption1="0.588ETH"
+          hint1="(~$1015.01)"
+          captionColor="white"
+          hintColor="grey"
+        />
       </Card>
-      <Card>
-        <span>Card4</span>
+      <Card xs={4} sm={4} md={4} lg={4}>
```
[the remark this answers is not in the document]
Comment (stackseeder) [reply src/components/common/CardRow/CardRow.stories.tsx]: It was intention to make the last card different from others but I've made a slight mistake. I've fixed it.

File: src/components/common/Card/CardContent.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+      variant={variant}
+      valueSize={valueSize}
+      labelWidth={labelWidth}
+      direction={direction}
+      captionColor={captionColor}
+      hintColor={hintColor}
+    >
+      <div>
+        <p>
+          {icon1} &nbsp;{label1}
```
[the remark this answers is not in the document]
Comment (stackseeder) [reply src/components/common/Card/CardContent.tsx]: I've fixed it.

**Comment by alfetopito** [comment]
@stackseeder I've merged the parent PR. Let me know once you addressed the conflicts to review it again

**Review by alongoni — APPROVED** [review]
Looks good!! :+1:

**Review by alfetopito — APPROVED** [review]

**Review by ramirotw — APPROVED** [review]

**Pull request merged**